include/mxnet/base.h

```diff
@@ -150,14 +154,14 @@ struct Context {
    * \return cpu::kDevMask or gpu::kDevMask
    */
   inline DeviceType dev_mask() const {
-    if (dev_type == kCPUPinned || dev_type == kCPUShared || dev_type == kNNP) return kCPU;
+    if (dev_type == kCPUPinned || dev_type == kCPUShared || dev_type == kNGraph) return kCPU;
```

Comment:
may give compile error if built without MXNET_USE_NGRAPH

Reply:
Made kNGraph always compile to avoid this, but throws an error if python attempts to initialize an nGraph context when mxnet wasn't compiled with nGraph.